coldfront/core/project/templates/project/project_request/savio/project_request_detail.html

```diff
@@ -76,7 +76,13 @@ <h2>
                     <strong>Timestamp:</strong>
                     {{ denial_reason.timestamp|iso8601_to_datetime }}
                   </p>
-                  <p>If you have any questions or concerns, please contact us at {{ support_email }}.</p>
+                    <p>
```

Comment:
I haven't checked, but is it the case that the user can see and click on the "Undeny" button, but they'll get an error?

I'd prefer that only superusers can see it.